core/prelude-editor.el

```diff
@@ -285,8 +285,8 @@ The body of the advice is in BODY."
 (require 'dired-x)
 
 ;; ediff - don't start another frame
-(require 'ediff)
-(setq ediff-window-setup-function 'ediff-setup-windows-plain)
+;; (require 'ediff)
+(setq-default ediff-window-setup-function 'ediff-setup-windows-plain)
```

Comment:
@bbatsov  Frankly, I guess it maybe about the variable scope.
The `ediff-window-setup-function` is custom variable, and I just try the 
```
(custom-set-variables '(ediff-window-setup-function 'ediff-setup-windows-plain))
```
It also resolve the issue, and the variable `ediff-window-setup-function` changed by `custom-set-variables` will relate to the graphic user interface. So `custom-set-variables` maybe a better solution.